tests/std/tests/P0355R7_calendars_and_time_zones_dates/test.cpp

```diff
@@ -281,6 +281,10 @@ constexpr void weekday_test() {
     assert(Sunday - Monday == days{6});
     assert(Sunday - Tuesday == days{5});
     assert(Wednesday - Thursday == days{6});
+
+    // GH-5153 "<chrono>: Undefined behaviour in weekday::weekday(const sys_days&)."

```

Comment:
I edited the issue title to make this self-explanatory, so I don't have to post a "please summarize the issue here" comment:
```suggestion
    // GH-5153 "<chrono>: integer overflow in weekday::weekday(sys_days::max())"
```